Source/carthage/Extensions.swift

```diff
@@ -89,8 +99,10 @@ extension Project {
 		let cartfileLock = "Cartfile.lock"
 		let carthageBuild = "Carthage.build"
 		let carthageCheckout = "Carthage.checkout"
-
-		let migrationMessage = "*** MIGRATION WARNING ***\n\nThis project appears to be set up for an older (pre-0.4) version of Carthage. Unfortunately, the directory structure for Carthage projects has since changed, so this project will be migrated automatically.\n\nSpecifically, the following renames will occur:\n\n  \(cartfileLock) -> \(CarthageProjectResolvedCartfilePath)\n  \(carthageBuild) -> \(CarthageBinariesFolderPath)\n  \(carthageCheckout) -> \(CarthageProjectCheckoutsPath)\n\nFor more information, see https://github.com/Carthage/Carthage/pull/224.\n"
+		
+		let formatting = colorOptions.formatting
+		
+		let migrationMessage = formatting.bulletin(string: "*** MIGRATION WARNING ***") + "\n\nThis project appears to be set up for an older (pre-0.4) version of Carthage. Unfortunately, the directory structure for Carthage projects has since changed, so this project will be migrated automatically.\n\nSpecifically, the following renames will occur:\n\n  \(cartfileLock) -> \(CarthageProjectResolvedCartfilePath)\n  \(carthageBuild) -> \(CarthageBinariesFolderPath)\n  \(carthageCheckout) -> \(CarthageProjectCheckoutsPath)\n\nFor more information, see " + formatting.URL(string: "https://github.com/Carthage/Carthage/pull/224") + ".\n"
```

Comment:
Can the formatter add these asterisks itself? It'd be nice to have that shared amongst all formatting of that type.